### PR TITLE
Bubble on single emulate subghz

### PR DESCRIPTION
**Pull request by Programistich**
**Background**

Hold to send infinity signal

**Changes**

* String res
* Colors for bubble
* Conditions for open and close bubble
* Popup with Text + Canvas; setup by coordinate

**Test plan**

Tap on emulate subghz button and see bubble
Hold on emulate subghz button and dont see bubble

### CODE REVIEW

File: components/keyscreen/emulate/src/main/java/com/flipperdevices/keyscreen/emulate/composable/common/ComposableEmulateButtonWithText.kt [hunk 1]
```diff
@@ -47,7 +48,7 @@ fun ComposableEmulateButtonWithText(
         )
 
         Row(
-            modifier = Modifier.padding(top = 4.dp),
+            modifier = Modifier.padding(top = 0.dp),
```
Comment (Programistich) [inline components/keyscreen/emulate/src/main/java/com/flipperdevices/keyscreen/emulate/composable/common/ComposableEmulateButtonWithText.kt:51]: Delete padding

File: components/keyscreen/emulate/src/main/java/com/flipperdevices/keyscreen/emulate/composable/common/ComposableHoldToSendInternal.kt [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Popup
+import com.flipperdevices.core.ui.theme.FlipperThemeInternal
+import com.flipperdevices.core.ui.theme.LocalPallet
+import com.flipperdevices.core.ui.theme.LocalTypography
+import com.flipperdevices.keyscreen.emulate.R
+
+private const val RADIUS = 15f
```
Comment (Programistich) [inline components/keyscreen/emulate/src/main/java/com/flipperdevices/keyscreen/emulate/composable/common/ComposableHoldToSendInternal.kt:44]: Rename constants

File: components/keyscreen/emulate/src/main/java/com/flipperdevices/keyscreen/emulate/composable/ComposableSubGhzSendButton.kt [hunk 3]
```diff
@@ -95,12 +103,23 @@ private fun ComposableActiveStateEmulateInternal(
...
+            val coordinate = it.positionInRoot()
+            positionYEmulateButton = coordinate.y.toInt()
+        },
         buttonActiveModifier = buttonActiveModifier,
         emulateProgress = (emulateButtonState as? EmulateButtonState.Active)?.progress,
         isActive = emulateButtonState is EmulateButtonState.Active
     )
+
+    // After single emulate close bubble
+    if (isBubbleOpen && emulateButtonState is EmulateButtonState.Inactive) isBubbleOpen = false
```
Comment (LionZXY) [inline components/keyscreen/emulate/src/main/java/com/flipperdevices/keyscreen/emulate/composable/ComposableSubGhzSendButton.kt:119]: You should store this value in persist storage

**Review by LionZXY — APPROVED** [review]
Good code!

**Pull request merged**